libbeat/metrics/process/process.go

```diff
@@ -258,6 +263,7 @@ func (procStats *ProcStats) getProcessEvent(process *Process) common.MapStr {
 			"norm": common.MapStr{
 				"pct": process.cpuTotalPctNorm,
 			},
+			"total_since_start": process.cpuSinceStart,
```

Comment:
* Is this a change that would also show up in Metricbeat?
* Cam we call this `total.value`? As far as I can see this is the main value we need to do all the calculations for the other values.
* Do we report NumCPU somewhere? If not, I think we should add it as it is required to make all calculations on the ES / KB side if needed.

Reply:
* Yes, it shows up in metricbeat. I need to add documentation for that new field.
* Yes, I will rename it.
* I will check it. If not, I will add it to the event.